packages/content/src/portal.ts

```diff
@@ -109,9 +109,8 @@ export function itemToContent(item: IItem): IHubContent {
  *
  * @param itemOrType an item or item.type
  */
-export function getItemHubType(itemOrType: IItem | string): HubType {
-  const itemType =
-    typeof itemOrType === "string" ? itemOrType : getType(itemOrType);
+export function getItemHubType(itemOrType: IItem): HubType {
```

Comment:
I think we still need to support string here, just pas it in as `{ type: itemOrType }` when it's a string and add this to  #305

Reply:
Done